src/main/resources/templates/showOrders.html

```diff
@@ -61,6 +61,18 @@
 <h1>Welcome to Ice Cream Up!</h1>
 <h2>Submitted Orders</h2>
 
+<table>
+<tr>
+<th>Order ID - </th>
+<th>Name - </th>
+<th>Address - </th>
+<th>Phone Number - </th>
+<th>Email Address - </th>
+<th>Date and Time Submitted - </th>
+<th>Comments</th>
+</tr>
+</table>
+
```

Comment:
This table header should help users understand the format that orders appear in.  It is a table so that it could be expanded in the future if you so choose.